.github/actions/cargo-install-upload-artifacts/action.yml

```diff
@@ -13,10 +13,8 @@ runs:
       run: |
         metadata="$(cargo metadata --format-version 1 --no-deps)"
 
-        package_name="$(jq -r '.packages[0].name' <<< "${metadata}")"
-        has_binaries="$(jq '.packages[0].targets | any(.kind | contains(["bin"]))' <<< "${metadata}")"
+        package_name="cross"
         echo "::set-output name=package-name::${package_name}"
-        echo "::set-output name=has-binaries::${has_binaries}"
```

Comment:
This does not remove the binary artifacts, it simply removes the check for if there is a binary or not.

This is done because rn, `packages[0]` is the xtask workspace member, but if we add another workspace member without a binary, and that happens to become the first package in the metadata, we'd silently not upload an artifact